projects/vdk-core/src/vdk/internal/builtin_plugins/termination_message/writer.py

```diff
@@ -56,20 +73,26 @@ def _write_termination_message(self, error_overall, user_error, configuration):
                         f'Writing termination message to file "{termination_message_output_file}"'
                     )
                 self._execute_termination_action(
-                    termination_action, error_overall, user_error
+                    self, file_util, error_overall, user_error, execution_skipped
                 )
-        except:
-            log.exception("Failed to write termination message.")
+        except Exception as e:
+            log.exception(e)
```

Comment:
Provide some context as to where and why this exception occured.
```suggestion
            log.exception("Failed to write termination message. See exception: ", e)
```